### PR TITLE
Ch2855 remove oauth creation steps

### PR DESCRIPTION
**Pull request by iain-b**
I've removed mentions of the user OAuth app (I don't think we mention the vouch app here ?). We could have a separate page in integrations about the github app ? there shouldn't be much to it though for the user so I thought what is in getting started was enough. I put a deprecation message on the github token page (but this might suit some users still?) and moved it to the bottom of the section.

https://deploy-preview-317--roadie.netlify.app/docs/integrations/github-org/

### CODE REVIEW

File: content/docs/getting-started/getting-started-for-admins/index.md [hunk 1]
```diff
@@ -27,22 +27,12 @@ In order to complete this guide, you will need the following things:
...
-
-We are working to contribute a fix upstream that will allow you to choose limited respositories, but for now please choose the “All repositories” option.
-
-### Step 1. Install the GitHub App
+Roadie needs to connect to GitHub in order to function. This can be setup by installing the Roadie GitHub App.
+Installing the GitHub App will allow Roadie to:
 
+1.  Access the YAML metadata files that Backstage needs to operate.
+2.  Read your GitHub organisation's users and team structure into backstage.  
+3.  Make requests from the browser on your behalf e.g. listing open pull requests.
```
Comment (iain-b) [inline content/docs/getting-started/getting-started-for-admins/index.md:35]: Technically they don't need to install the app into their org to do this they just need to approve it to access their account.

**Comment by dtuite** [comment]
@iain-b I'm confused by something. I see this page in the preview but not in the diff. I must be missing something simple here.

https://deploy-preview-317--roadie.netlify.app/docs/integrations/github-org/

**Comment by iain-b** [comment]
https://github.com/RoadieHQ/marketing-site/pull/317/files#diff-e2fad3e71426191eeae7be02e031541ba5d079a8d3c6115225bc599409b9e060 ? 

It was there before so no change to it in the sidebar (though it's first because others have moved).

**Review by dtuite — APPROVED** [review]
LOVE to see us deleting steps from the onboarding docs. It's a good proxy for end user product complexity IMO.

**Comment by dtuite** [comment]
@iain-b I'm going to merge this for you now because I'm about to send a tenant to someone and it's clearer for them if there are no OAuth steps in the instructions.

Even though the work to use the GitHub app for OAuth is not merged yet, I'm manually creating individual OAuth apps for each company in the Roadie developer settings, so they don't need the OAuth steps in the docs.

**Pull request merged**